### DISCUSSION TITLE
Return created and updated

### DISCUSSION THREAD

**Discussion by phstc** [sensedeep/dynamodb-onetable#331] [General]
This is a `[question]`.

I want `createdAt` and `updatedAt` to be available in the TypeScript interface I have for my models, besides renaming them to `createdAt` and `updatedAt`.

One way I found to implement that is to redundantly define  `createdAt` and `updatedAt` as `type: Date` properties. 

```
 /* ... */
  models: {
    Test: {
     /* ... */
      createdAt: { type: Date, required: true },
      updatedAt: { type: Date, required: true },
    },
  } as const,
  params: {
    isoDates: true,
    timestamps: true,
    createdField: 'createdAt',
    updatedField: 'updatedAt',
  },
};

export type TestType = Entity<typeof Schema.models.Test>;
```

Is that the correct way? Should the type be `Date` or `Number`?

**Comment by mobsense** [comment]
Yes, you can add to your schema if you need.

Type should be Date.
